app/controllers/users_controller.rb

```diff
@@ -0,0 +1,43 @@
+class UsersController < ApplicationController
+
+	def index
+		@users = User.all
+	end
+
+	def new
+		@user = User.new
+	end
+
+	def create
+		@user =	User.new(user_params)
+
+		if @user.save
+			session[:user_id] = @user.id
```

Comment:
@saki4567 
Use this method `log_in @user`
Related to this: https://github.com/framgia831/rails-elearning-haruna/pull/2/files#diff-c70c347141b21b6f03e13ad752ecca26R5